ui/app/src/components/PluginDetailsView/PluginDetailsView.tsx

```diff
@@ -315,8 +338,16 @@ export default function PluginDetailsView(props: PluginDetailsViewProps) {
                 />
               </Alert>
             )}
-
             <Typography variant="body1">{description}</Typography>
+            {markdown && <Typography component="div" dangerouslySetInnerHTML={{ __html: markdown }} />}
+            {link && <Link href={link} />}
+            {markdownError && (
+              <ErrorState message={formatMessage(messages.markdownError)}>
+                <Typography>
+                  <Link href={plugin.documentation}>{plugin.documentation}</Link>
+                </Typography>
+              </ErrorState>
```

Comment:
Let's not make too much fuss about the error. Just stick to a message:

Unable to render documentation. Visit {link} to view.